tools/kubeblocks_helm/pika-master-slave/templates/componentdefinition-pika.yaml

```
@@ -25,30 +25,32 @@ spec:
-             done
-        preCondition: ComponentReady
+            - /bin/bash
+            - -c
+            - |
+              replication_info=$(redis-cli -h 127.0.0.1 -p 9221 info replication)
+              role=$(echo "${replication_info}" | awk -F':' '/^role:/ {print $2}' | tr -d '[:space:]')
+              if [ "$role"x = "master"x ]; then
+                echo -n "master"
+              else
+                echo -n "slave"
+              fi  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)